sdk/communication/communication-sms/test/public/utils/assertHelpers.ts

```diff
@@ -13,6 +13,7 @@ export const assertIsSuccessResult = (
   assert.isString(actualSmsResult.messageId);
   assert.isTrue(actualSmsResult.successful);
   assert.notExists(actualSmsResult.errorMessage, "no error message for success");
+  assert.equal((actualSmsResult as any).repeatabilityResult, "accepted"); // internal field
```

Comment:
doing this so I don't need to import the internal result, which would make min-max testing a lot tougher